### ISSUE TITLE
Issue in Safari

### ISSUE THREAD

**Issue by sjsnider** [web-dave/ngx-img-cropper#9]
For some reason the cropper isn't making what is to be the cropped part of the image lighter in safari, but it works just fine in Chrome.  First image is from Chrome, 2nd one is from Safari

![image](https://user-images.githubusercontent.com/1031722/36696866-49bc6c0c-1afa-11e8-8ff9-6e24f3af1861.png)

![image](https://user-images.githubusercontent.com/1031722/36696955-89b98d30-1afa-11e8-880e-1504266d2a32.png)

```
    this.cropperSettings = new CropperSettings();
    this.cropperSettings.width = 140;
    this.cropperSettings.height = 140;
    this.cropperSettings.keepAspect = false;

    this.cropperSettings.croppedWidth = 140;
    this.cropperSettings.croppedHeight = 140;

    this.cropperSettings.canvasWidth = 500;
    this.cropperSettings.canvasHeight = 300;

    this.cropperSettings.minWidth = 100;
    this.cropperSettings.minHeight = 100;

    this.cropperSettings.rounded = true;
    this.cropperSettings.minWithRelativeToResolution = false;

    this.cropperSettings.cropperDrawSettings.strokeColor = 'rgba(255,255,255,.6)';
    this.cropperSettings.cropperDrawSettings.strokeWidth = 2;
    this.cropperSettings.noFileInput = true;
    this.cropperSettings.fileType = 'image/png';
```

**Comment by sjsnider** [comment]
just following up real quick @web-dave , do you think you'll get a chance to check this out?  Thanks

**Comment by web-dave** [comment]
I'll check it today. I'm on my way to my office.

**Comment by web-dave** [comment]
fixed in 0.10.2
